package.json

```diff
@@ -258,5 +262,11 @@
     "sqlite-orm",
     "sql-server",
     "sql-server-orm"
-  ]
+  ],
+  "lint-staged": {
```

Comment:
We should avoid adding tool config to `package.json` - this file should only be used by the package manager (`npm` in our case).

`lint-staged` config should go into `.lintstagedrc.json`.